HIP-xxxx.md

```diff
@@ -0,0 +1,163 @@
+
+## TXT Standards Definition
+
+For the above to work we need standards so any website can properly render the TXT records. The records should be flexible, short and easy to understand. Since it's impossible to validate the data via HSD. Therefore TXT records should be provided mostly without any prefix like "mailto:", "tel:" or "https://www.twitter.com/".
+
+The goal is to let the app or website decide how to render the information. It also decreases the data saved on-chain.
+
+#### Domain Parking
+
+The parking TXT entry indicates that the domain is for sale.
+
+```
+TXT parking=[String]
+```
+
+#### Contact or Profile Information
+
+```
+TXT email=[Email]
+TXT phone=[Tel]
+```
+
+```
+TXT discord=[String]
+TXT facebook=[String]
+TXT github=[String]
+TXT instagram=[String]
+TXT linkedin=[String]
+TXT telegram=[String]
+TXT tiktok=[String]
+TXT twitter=[String]
+TXT weixin=[String]
+TXT whatsapp=[Tel]
+TXT youtube=[String]
+```
+
+#### Avatar
+
+```
+TXT avatar=[ URL | skylink | ipfsLink ]
+```
+
+#### Domain Caption
+
+A caption is especially useful for identities or as a sales pitch. It can also be used to add more context to a domain (e.g. NFT's).
+
+```
+TXT caption=[String]
+```
+
+#### Payment / Wallet Alias
+
+Although [HIP-0002](https://github.com/handshake-org/HIPs/blob/master/HIP-0002.md) is a much better approach, TXT entries could also be used to provide static wallet information.
+
+```
+TXT wallet=[Chain]:[Address]
+```
+
+## Valid TXT Standard Examples
+
+#### Parking Examples
+
+```
+TXT parking
+TXT parking=10000 HNS
+TXT parking=USD 1000
+TXT parking=USD 999.99
+TXT parking=999,99 EUR
+TXT parking=$199 (Old price $329)
+TXT parking=Please send an offer via email
+```
+
+Every string is valid, but usually a price is provided. Data providers should be aware that prices could be devided by a dot or a comma and can have decimal points.
+
+No matter what data is provided, as long as the parking TXT entry is added, the domain will be seen as "For Sale". It's up to the site owner or app to make sense of the data. Remember: It's impossible to validate the data via HSD.
+
+#### Contact Information Examples
+
+```
+TXT email=hello@handshake.org
+TXT phone=+1(310)3015800
+```
+
+The phone number should be prefixed with a plus sign and country code. Hyphens, dots and empty spaces are allowed.
+
+```
+TXT discord=handshake#1234
+TXT facebook=handshake
+TXT github=handshake
+TXT instagram=handshake
+TXT linkedin=handshake
+TXT telegram=handshake
+TXT tiktok=handshake
+TXT twitter=handshake
+TXT weixin=DDvRyffEabcdefGh123e
+TXT whatsapp=907123987234
+TXT youtube=handshake
+```
+
+For most social media profiles it's enough to provide the handle. WeChat (weixin) needs a chat ID. WhatsApp links will only work without a prefix, therefore it's best to omit them. We can always add or remove networks if needed.
+
```

Comment:
I'm particularly against proposals that are too open. There are too many scenario to be covered.
IMO, it would be better to have an HIP specific to each subject: NFT, listing, parking and so on.

Reply:
Agreed. I will split them into 3 proposals